ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -496,6 +495,71 @@ class SignalSpec: QuickSpec {
 			}
 		}
 
+		describe("Signal.merge") {
+			it("should emit values from all signals") {
+				let (signal1, observer1) = Signal<Int, NoError>.pipe()
+				let (signal2, observer2) = Signal<Int, NoError>.pipe()
+
+				let mergedSignals = Signal.merge([signal1, signal2])
+
+				var lastValue: Int?
+				mergedSignals.observeNext { lastValue = $0 }
+
+				expect(lastValue).to(beNil())
+
+				observer1.sendNext(1)
+				expect(lastValue) == 1
+
+				observer2.sendNext(2)
+				expect(lastValue) == 2
+
+				observer1.sendNext(3)
+				expect(lastValue) == 3
+			}
+
+			it("should not stop when one signal completes") {
```

Comment:
Actually, 
> Would it make sense to write a similar tests for when one of the signals sends an error?

That test would verify that no values are emitted after an error? That's basically testing the `Signal` contract, which should be tested elsewhere :)